### PR TITLE
feat: add CTA to ory network

### PR DESCRIPTION
**Pull request by LBBO**
This CTA differs a bit from the design in the interest of saving time.

### CODE REVIEW

**Comment by aeneasr** [comment]
Isn't this missing some button i can click on?

**Comment by LBBO** [comment]
Fixed. Apparently the docusaurus production build just deletes `@container` queries, lol. But there ended up being no necessity for them, so I just exchanged them for equivalent media queries.

**Review by OryTiki — CHANGES_REQUESTED** [review]
Still need some work on the button styling or content to ensure most viewports look good. 

image

File: src/components/OryNetworkCta/ory-network-cta.css [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  text-decoration: none;
+}
+
+.ory-network-cta__inline-get-started {
+  display: block;
+}
+
+@media (min-width: 1070px) {
+  .ory-network-cta__button {
+    display: block;
```
Comment (jfcurran) [inline src/components/OryNetworkCta/ory-network-cta.css:94]:
```suggestion
    display: block;
    text-align: center;
```

**Review by jfcurran — CHANGES_REQUESTED** [review]
the button text should be centered, this change will do the trick

**Review by vinckr — COMMENTED** [review]
lgtm. 

only nit is that its not present in the "self-hosting" category, is that on purpose?

**Comment by aeneasr** [comment]
@vinckr it's only visible if there is a table of contents on the right side: https://docs-fgqd0n2bm-ory.vercel.app/docs/ecosystem/configuring

**Pull request merged**